File: src/main/java/com/gangofconnectfour/powerfourservice/repository/MultiGameRepository.java
package com.gangofconnectfour.powerfourservice.repository;

import com.gangofconnectfour.powerfourservice.model.MultiGame;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MultiGameRepository extends JpaRepository<MultiGame, Long> {
}

File: src/main/kotlin/com/gangofconnectfour/powerfourservice/repository/MultiGameRepository.kt
package com.gangofconnectfour.powerfourservice.repository

import com.gangofconnectfour.powerfourservice.model.MultiGame
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MultiGameRepository : JpaRepository<MultiGame, String> {
}